docs/api/api/api.txt

```diff
@@ -712,6 +712,22 @@ Parameters:
 XmlResult: status
 
 
+POST /source/<project>/<package>?cmd=fork&scmsync=url
+
+  Creating a scmsync based fork of a package. The package sources must be
+  specified via the scmsync paramater. A project will be created or an
+  existing one modified to provide the repository setup.
+  The source project must exists, the source package may not exist.
+  The default target project is by default below the home:$user:branches
+  namespace.
```

Comment:
This should also include what is taken from the source project and how this relates to the branch command.

```suggestion
  Create a scmsync based fork of a package. The package sources must be
  specified via the scmsync parameter. A project will be created or an
  existing one modified to provide the repository setup.
  The source project must exists, the source package may not exist.
  The default target project is by default below the home:$user:branches
  namespace.
```